android/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.gitignore eclipse stuff

Reply:
same as above ... not sure where this came from